autokeras/hypermodel/hyper_block.py

```diff
@@ -41,9 +41,20 @@ def build(self, hp, inputs=None):
         output_node = Flatten().build(hp, output_node)
 
         for i in range(hp.Choice('num_layers', [1, 2, 3], default=2)):
-            output_node = tf.keras.layers.Dense(hp.Choice('units_{i}'.format(i=i),
-                                                          [16, 32, 64],
-                                                          default=32))(output_node)
+            if hp.Choice('layer_category', ['Dense','BN','Dropout','Activation'], default='Dense') == 'Dense':
+                output_node = tf.keras.layers.Dense(hp.Choice('units_{i}'.format(i=i),
+                                                              [16, 32, 64],
+                                                              default=32))(output_node)
+            elif hp.Choice('layer_category', ['Dense','BN','Dropout','Activation'], default='Dense') == 'BN':
+                # TODO: Insert BN layer
+                output_node = tf.keras.layers.BatchNormalization()(output_node)
+            elif hp.Choice('layer_category', ['Dense','BN','Dropout','ACtivation'], default='Dense') == 'Dropout':
+                # TODO: Insert Dropout layer
+                output_node = tf.keras.layers.Dropout(rate=hp.Choice('dropout_rate',[0.1, 0.3, 0.5],default=0.5))(output_node)
+            else:
+                # TODO: Insert Activation layer
```

Comment:
TODO means something to be done. I think it is already done in the code. So we don't need to use all the TODOs in here